mail/dovecot/Portfile

```diff
@@ -3,10 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        dovecot core 2.3.10.1
+github.setup        dovecot core 2.3.11.3
 name                dovecot
 revision            0
-epoch               20060722
+epoch               20060723
```

Comment:
Don't change the epoch since the version number has advanced in normal sequence.